README.md

```diff
@@ -121,7 +120,18 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
 }
 ```
 
-### Animations
+##### Custom slides example
+
+One may also define custom slides as seen in the example project:
+ * Copy the class **SampleSlide** from my [example project](https://github.com/PaoloRotolo/AppIntro/blob/master/example/src/main/java/com/github/paolorotolo/appintroexample/SampleSlide.java).
+ * Add a new slide with ```addSlide(SampleSlide.newInstance(R.layout.your_slide_here));```

```

Comment:
This was actually just copy and pasted from the old readme, but I'll address it.